EnglishTranslation-master/ETC.tsv

```diff
@@ -13718,7 +13718,7 @@ ETC_20150729_013719	{nl}{nl}The lower part displays the total carry weight of yo
 ETC_20150729_013720	Available quests can be found in the Quest window.
 ETC_20150729_013721	Press {img F5 40 40} to view available quests {nl}or quests that are in progress.
 ETC_20150729_013722	{nl}{nl}You can restart abandoned quests by pressing the 
-ETC_20150729_013723	 button. Some quests cannot be abandoned.
+ETC_20150729_013723	 button appear on quests that cannot be abandoned.
```

Comment:
i don't think this change is necessary if this line follows up the line directly before it?